grub/02_coreos_firstboot

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+exec tail -n +3 $0
+set first_boot=""
+# Determine if this is a first boot.
+if [ -f "/boot/coreos/first_boot" ]; then
+    set first_boot="coreos.first_boot=detected"
```

Comment:
set first_boot=coreos.ignition-first-boot?

Reply:
I think @Rahuls0720 is pointint out that we are setting `first_boot="coreos.first_boot=detected` here but checking `ConditionKernelCommandLine=coreos.ignition-first-boot` in the `ignition-quench.service` below